src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/utils.ts

```diff
@@ -123,3 +123,14 @@ export const areDefaultFiltersSelected = (
   if (!transportModes || transportModes.length === 0) return false;
   return transportModes.every((tm) => tm.selectedAsDefault === tm.selected);
 };
+
+export const isVenue = (location: Location): boolean => {
+  return location.resultType === 'search' && location.layer === 'venue';
+};
+
+export const isGroupOfStopPlaces = (location: Location): boolean => {
+  return (
+    location.resultType === 'search' &&
+    location.category[0] === 'GroupOfStopPlaces'
```

Comment:
`'GroupOfStopPlaces'` should be replaced by the enum `GROUP_OF_STOP_PLACES` directly

Reply:
Ok, will fix👍